### PR TITLE
Added photo and information

### PR DESCRIPTION
**Pull request by Denisa03**
Added photo and information.

### CODE REVIEW

**Comment by roonyh** [comment]
Please follow instructions here https://github.com/fossasia/fossasia.github.io/wiki/Installing-GitHub-And-Adding-Images

File: _data/students.json [hunk 1]
```diff
@@ -404,5 +404,17 @@
     },
     "imgName": "Riyaz.jpg",
     "name": "Riyaz Shaikh"
-}    
+},
+{
+    "social": {
+        "github": "Denisa03",
+        "twitter": "Denisa Pîrlea"
```
Comment (roonyh) [inline _data/students.json]: Need a comma at the end of this line. See other entries.

File: index.html [hunk 2]
```diff
@@ -344,6 +344,15 @@ <h1 class="section__title">Facebook</h1>
...
+"social": {
+        "github": "Denisa03",
+        "twitter": "Denisa Pîrlea",
+        "facebook": "Pîrlea Denisa",
+        "mail": "denisapirlea03@gmail.com",
+        "link": " http://nisa03.edublogs.org ",
+        
+    },
+    "imgName": "Denisa03.jpg",
+    "name": "Denisa"
```
Comment (roonyh) [inline index.html]: Please remove this content. You don't need to add anything in index.html.

**Comment by mariobehling** [comment]
Hi Denisa, it looks like your commit is outdated unfortunately. Please help us and base your work on the current files. Thank you so much!

**Comment by Denisa03** [comment]
what do i have to do?

**Comment by roonyh** [comment]
This should help. http://stackoverflow.com/questions/17182624/contributing-to-project-on-github-how-to-rebase-my-pull-request-on-top-of-mast

**Comment by Denisa03** [comment]
changed something??

**Comment by tradzik** [comment]
@Denisa03 It seems you didn't make any changes. If you need help with rebasing you can read this article - https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request. It shows, on examples, how to do that :-) Thank you for your edits!

**Pull request closed without merging**